### DISCUSSION TITLE
Github Copilot plugin for JetBrains PyCharm EAP 2023.2 - 232.5150.120

### DISCUSSION THREAD

**Discussion by johnkennyuk** [community/community#55640] [Copilot Conversations]
### Select Topic Area

Bug

### Body

**Describe the bug**
Github Copilot Plugin 1.2.6.25.95-nightly is incompatible with jetbrains Pycharm EAP 2023.2 - 232.5150.120

**To reproduce**
Install PyCharm EAP 2023.2 and start, If PyCharm was upgraded the notification immediately shows if using a clean instance then after attempting to install the plugin via marketplace or via settings-sync the warning will display after a restart of PyCharm.

**Expected behavior**
PyCharm should start and not warn about the incompatibilities and the plugin will load.

**Screenshots**
![image](https://github.com/community/community/assets/64958468/f6f9f4c8-b003-4e51-ba53-2105b0aed632)
![image](https://github.com/community/community/assets/64958468/93163801-2b56-4d44-b332-ad9245455b85)
![image](https://github.com/community/community/assets/64958468/537605e6-f242-4b87-9222-fae184e8cfd9)

**Your Environment**
- OS: Windows 11 Ent 25357
- JetBrains product: PyCharm EAP
- JetBrains product version: 2023.2 - 232.5150.120
- Github Copiliot plugin version: 1.2.6.25.95-nightly

**Comment by Scott-G** [comment]
This is an issue with PHPStorm 2023.2 EAP too, so presumably will be the same across all Jetbrains EAPs?
Assuming no compatibility issues were introduced, this should be a simple revision bump. Either way, can we have an ETA?

**Comment by xiandong79** [comment]
any progress?

**Comment by q42jaap** [comment]
CoPilot nightly should target the EAP so an EAP user can manually install the nightly of the plugin. Now there is no way to experience even the bleading edge of copilot in EAP versions of Jetbrains IDEs.
